### DISCUSSION TITLE
请问我应该怎样设置我的gitlens显示为中文？

### DISCUSSION THREAD

**Discussion by Alex-zry-2333** [gitkraken/vscode-gitlens#3188] [Q&A] [answered]
我已经把vscode的编码改成了zh-cn
但是gitlens显示的内容还是以英文显示的。
另外在gitlens的插件设置界面我也没有找到语言设置的地方。
请问我应该怎样操作才能让我的gitlens以中文的方式显示内容？这样方便向孩子教学。

**Comment by jkelroy** [comment, marked as answer]
Hello @Alex-zry-2333, there is an open feature request for i18n support here: https://github.com/gitkraken/vscode-gitlens/issues/577
